go-owasm/src/lib.rs

```diff
@@ -151,7 +161,7 @@ fn require_mem_range(max_range: usize, require_range: usize) -> Result<(), Error
     return Ok(());
 }
 
-fn run(code: &[u8], gas_limit: u32, span_size: i64, is_prepare: bool, env: Env) -> Result<(), Error> {
+fn run(code: &[u8], gas_limit: u32, span_size: i64, is_prepare: bool, env: Env) -> Result<u32, Error> {
```

Comment:
Just a question, if we. return Result<RunOutput, Error> like other function will make more. sense?